front/dropdown.common.form.php

```diff
@@ -134,10 +134,10 @@
 } else {
    $dropdown->displayHeader();
 
-   if (!isset($options)) {
-      $options = [];
-   }
-   $options['id'] = $_GET["id"];
+   $options = [
+      'id'           => $_GET['id'],
+      'formoptions'  => "data-track-changes=true"
+   ];
```

Comment:
`$options` may be defined by the front dropdown file including current file.
```suggestion
   if (!isset($options)) {
         $options = [];
   }
   $options['id'] = $_GET["id"];
   $options['formoptions'] = ($options['formoptions'] ?? '') . " data-track-changes=true";
```